recipes/sqlalchemy-migrate/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.9.7" %}
+
+package:
+  name: sqlalchemy-migrate
+  version: {{ version }}
+
+source:
+  fn: sqlalchemy-migrate-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/s/sqlalchemy-migrate/sqlalchemy-migrate-{{ version }}.tar.gz
+  md5: fd586f77b25f905d56d1847fc1f9c68b
+
+build:
+  number: 0
+  entry_points:
+    - migrate = migrate.versioning.shell:main
+    - migrate-repository = migrate.versioning.migrate_repository:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr >=1.3
+
+  run:
+    - python
+    - "sqlalchemy >=0.7.8,!=0.9.5"
+    - six >=1.7.0
+    - decorator
+    - sqlparse
+    - tempita >-0.4
```

Comment:
Yes, yes it should.  None of the tools ever balked at that - I guess they were doing "greater than a negative number"?